.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug Report
+about: Anything that does not work as expected is a bug, including code, docs and
+  tests.
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+#### Description
+<!-- A clear and concise description of what the bug is. -->
+
+#### Repro Steps
+<!-- Steps to reproduce the behavior. -->
+
+#### Expected behavior
+<!-- A clear and concise description of what you expected to happen. -->
+
+#### Environment (please complete the following information):
+ - OS: [e.g. Mac OS X 10.4]
+ - Browser and Version [e.g. Chrome 71.0]
+ - Deck.gl Version [e.g. 6.3.0]
+ - GPU [e.g. AMD Radeon Pro 560X, see (http://webglreport.com)[http://webglreport.com/?v=2]]
+
+#### Logs
+<!-- Check the browser console for any relevant errors or warnings.  
+For issues in deck.gl examples, please provide detailed logs by typing `luma.log.priority=2` and `deck.log.priority=2` in browser console. -->
+
+#### Code Example
```

Comment:
I think this could be merged with `repro steps`. We can suggest something like:

```
Providing the following could help us resolve this issue faster:
- A Codepen [example](https://codepen.io/vis-gl) that reproduces the behavior
- A sample of your dataset
```